wagtail/project_template/home/templates/home/welcome_page.html

```diff
@@ -0,0 +1,380 @@
+{% templatetag openblock %} load i18n {% templatetag closeblock %}
+<link rel="stylesheet" type="text/css" href="/static/admin/css/fonts.css">
+<style type="text/css">
+  body, main {
+    margin: 0 auto;
+  }
+  .body, .tip {
+    stroke: #fff;
+  }
+  html {
+    line-height: 1.15;
+    -ms-text-size-adjust: 100%;
+    -webkit-text-size-adjust: 100%;
+    box-sizing: border-box;
+  }
+  footer, header, main {
+    display: block;
+  }
+  a {
+    background-color: transparent;
+    -webkit-text-decoration-skip: objects;
```

Comment:
What does this do? Can we use a standard property name instead?

Reply:
The desired behavior (having underlines "skip over" letter descenders) is now the default in all browsers that support this property. Removed in 959d41b.